app/controllers/welcome_controller.rb

```diff
@@ -1,6 +1,7 @@
 class WelcomeController<ApplicationController
 
   def index 
+    @featured_podcasts = Podcast.order('RANDOM()').limit(3)
```

Comment:
Very exciting, this will be cool.

Reply:
Could add a background worker in the future

Reply:
This RANDOM flag is awesome